core/primitives/src/crypto.rs

```diff
@@ -32,6 +32,7 @@ use regex::Regex;
 use base58::{FromBase58, ToBase58};
 #[cfg(feature = "std")]
 use std::hash::Hash;
+extern crate zeroize;
```

Comment:
interested to know why this is now needed? I thought this is no longer needed. 

also had a look and we should probably remove all `extern crate`s?